manager/src/main/java/org/openremote/manager/gateway/GatewayConnector.java

```diff
@@ -258,11 +259,12 @@ protected void getCapabilities() {
                     }
                 }
                 tunnellingSupported = response != null && response.isTunnelingSupported();
-                this.gatewayVersion = response.getVersion();
+                this.gatewayVersion = Objects.requireNonNull(response).getVersion();
                 this.tunnelTimeoutManagementSupported = response.isTunnelTimeoutManagementSupported();
 
-                if(!Objects.equals(getGatewayVersion(), GatewayCapabilitiesResponseEvent.CURRENT_VERSION)){
-                    LOG.warning("Gateway Version is not up to date, latest version is "+GatewayCapabilitiesResponseEvent.CURRENT_VERSION + ", Gateway version is " + getGatewayVersion() + ". GatewayId: " + gatewayId);
+                String currentManagerVersion = StatusResourceImpl.getManagerVersion();
+                if(!Objects.equals(getGatewayVersion(), currentManagerVersion)) {
```

Comment:
This logic is not correct we might increase the manager version but have no changes to the Gateway API so we instead need to keep a latest compatible version field to track this.